client/gutenberg/editor/controller.js

```diff
@@ -153,83 +53,30 @@ function getPressThisData( query ) {
 }
 
 export const post = ( context, next ) => {
-	//see post-editor/controller.js for reference
+	// See post-editor/controller.js for reference.
 
-	const uniqueDraftKey = uniqueId( 'gutenberg-draft-' );
 	const postId = getPostID( context );
 	const postType = determinePostType( context );
-	const isDemoContent = ! postId && has( context.query, 'gutenberg-demo' );
 	const jetpackCopy = parseInt( get( context, 'query.jetpack-copy', null ) );
 
-	//check if this value is an integer
+	// Check if this value is an integer.
 	const duplicatePostId = isInteger( jetpackCopy ) ? jetpackCopy : null;
 
-	if ( config.isEnabled( 'calypsoify/iframe' ) ) {
-		const state = context.store.getState();
-		const siteId = getSelectedSiteId( state );
-		const pressThis = getPressThisData( context.query );
-
-		// Set postId on state.ui.editor.postId, so components like editor revisions can read from it.
-		context.store.dispatch( { type: EDITOR_START, siteId, postId } );
+	const state = context.store.getState();
+	const siteId = getSelectedSiteId( state );
+	const pressThis = getPressThisData( context.query );
 
-		context.primary = (
-			<CalypsoifyIframe
-				postId={ postId }
-				postType={ postType }
-				duplicatePostId={ duplicatePostId }
-				pressThis={ pressThis }
-			/>
-		);
+	// Set postId on state.ui.editor.postId, so components like editor revisions can read from it.
+	context.store.dispatch( { type: EDITOR_START, siteId, postId } );
```

Comment:
I think we don't have to dispatch this anymore.

Reply:
We still need this for revision integrations. Please see https://github.com/Automattic/wp-calypso/pull/30962 for more details.

Reply:
👍